sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2_DetectLanguageBatch.cs

```diff
@@ -13,34 +13,71 @@ namespace Azure.AI.TextAnalytics.Samples
     public partial class TextAnalyticsSamples
     {
         [Test]
-        public void DetectLanguageBatch()
+        public void DetectLanguageBatchAdvanced()
         {
             string endpoint = Environment.GetEnvironmentVariable("TEXT_ANALYTICS_ENDPOINT");
             string subscriptionKey = Environment.GetEnvironmentVariable("TEXT_ANALYTICS_SUBSCRIPTION_KEY");
 
             // Instantiate a client that will be used to call the service.
             var client = new TextAnalyticsClient(new Uri(endpoint), subscriptionKey);
 
-            var inputs = new List<string>
+            var inputs = new List<DetectLanguageInput>
             {
-                "Hello world",
-                "Bonjour tout le monde",
-                "Hola mundo",
-                ":) :( :D"
+                new DetectLanguageInput("1")
+                {
+                     CountryHint = "us",
+                     Text = "Hello world",
+                },
+                new DetectLanguageInput("2")
+                {
+                     CountryHint = "fr",
+                     Text = "Bonjour tout le monde",
+                },
+                new DetectLanguageInput("3")
+                {
+                     CountryHint = "es",
+                     Text = "Hola mundo",
+                },
+                new DetectLanguageInput("4")
+                {
+                     CountryHint = "us",
+                     Text = ":) :( :D",
```

Comment:
;-)

Reply:
I took it from their docs!  It's currently a bug in the service that it returns English for this and not Unknown.